src/main/java/seedu/address/ui/SummaryStatsWindow.java

```diff
@@ -95,6 +102,22 @@ public void focus() {
         getRoot().requestFocus();
     }
 
+    private void updateTotalCountOfPersons() {
+        totalPerson = logic.getTotalPersons();
+        String output = SUMMARYSTATS_MESSAGE_COUNT + " " + Integer.toString(totalPerson);
+        summaryMessageCountLabel.setText(output);
+    }
+
+    private void updateTotalOwingsofPersons() {
+        totalOwings = logic.getTotalOwings();
+        String output = SUMMARYSTATS_MESSAGE_OWING + " " + Double.toString(totalOwings);
+        summaryMessageOwingsLabel.setText(output);
+    }
+
+    private void updateSummaryStats() {
```

Comment:
great update methods to constantly update summary stats when new person/payment is added or removed.